features/build/build_controller.rb

```diff
@@ -0,0 +1,29 @@
+require_relative "../../shared/authenticated_controller_base"
+require_relative "./build_websocket_backend"
+require "pathname"
+
+module FastlaneCI
+  # Controller for a single project view. Responsible for updates, triggering builds, and displaying project info
+  class BuildController < AuthenticatedControllerBase
+    HOME = "/projects/*/builds"
+
+    use(FastlaneCI::BuildWebsocketBackend)
+
+    get "#{HOME}/*" do |project_id, build_id|
+      project = self.user_project_with_id(project_id: project_id)
+      build = project.builds.find { |b| b.sha == build_id } # TODO: We need a build ID, sha isn't enough
```

Comment:
This should be just the number?

Reply:
Yes, once we resolved the question/proposal around having a unique Build ID